examples/bri-1/base-example/test/e2e.spec.ts

```diff
@@ -168,18 +189,24 @@ describe('baseline', () => {
           assert(inviteToken, 'invite token should not be null');
         });
 
-        describe('alice', async () => {
+        describe('alice', function () {
           before(async () => {
-            await app.requireWorkgroupContract('erc1820-registry');
-            await app.requireWorkgroupContract('organization-registry');
-            await aliceApp.acceptWorkgroupInvite(inviteToken, app.getWorkgroupContracts());
+            await bobApp.requireWorkgroupContract('erc1820-registry');
+            await bobApp.requireWorkgroupContract('organization-registry');
+            await aliceApp.acceptWorkgroupInvite(inviteToken, bobApp.getWorkgroupContracts());
           });
 
-          shouldBehaveLikeAnInvitedWorkgroupOrganization(aliceApp);
-          shouldBehaveLikeAWorkgroupOrganization(aliceApp);
+          describe(`invited workgroup organization: "${aliceCorpName}"`, shouldBehaveLikeAnInvitedWorkgroupOrganization.bind(this));
+          describe(`workgroup organization: "${aliceCorpName}"`, shouldBehaveLikeAWorkgroupOrganization.bind(this));
+          describe(`workgroup counterparty: "${aliceCorpName}"`, shouldBehaveLikeAWorkgroupCounterpartyOrganization.bind(this));
+        });
+
+        describe('counterparties post-onboarding', function () {
+          before(async () => {
+            this.ctx.app = bobApp;
+          });
 
-          shouldBehaveLikeAWorkgroupCounterpartyOrganization(bobApp);
-          shouldBehaveLikeAWorkgroupCounterpartyOrganization(aliceApp);
+          describe(bobCorpName, shouldBehaveLikeAWorkgroupCounterpartyOrganization.bind(this));
```

Comment:
To establishing consistency in naming of tests:
`bobCorpName` --> `workgroup counterparty: "${bobCorpName}"`